src/block/Beacon.php

```diff
@@ -23,11 +23,209 @@
 
 namespace pocketmine\block;
 
+use InvalidArgumentException;
+use pocketmine\block\inventory\BeaconInventory;
+use pocketmine\block\tile\Beacon as TileBeacon;
+use pocketmine\data\bedrock\EffectIdMap;
+use pocketmine\entity\effect\Effect;
+use pocketmine\entity\effect\EffectInstance;
+use pocketmine\entity\effect\VanillaEffects;
+use pocketmine\item\Item;
+use pocketmine\item\ItemTypeIds;
+use pocketmine\math\Vector3;
+use pocketmine\player\Player;
+use function array_merge;
+
 final class Beacon extends Transparent{
 
+	public const MIN_LEVEL_BEACON = 1;
+	public const MAX_LEVEL_BEACON = 4;
```

Comment:
Inclunding `BEACON` in the constant name seems redundant since it is on the `Beacon` class